android/app/src/main/java/org/mozilla/magnet/notifications/ImageNotification.java

```diff
@@ -53,7 +53,7 @@ protected void onPostExecute(Map<String,Bitmap> bitmaps) {
                 .setSmallIcon(R.drawable.ic_stat_notify)
                 .setAutoCancel(true)
                 .setDeleteIntent(dismissIntent)
-                .setContentIntent(createLaunchIntent());
+                .setContentIntent(createItemDeepLinkIntent(mItem.getUrl()));
```

Comment:
just bikesheeding a bit, should the name should be something like `getItemDeepLinkIntent`?
Not a strong opinion, but the name seems too large for me.